api/graphql/schema.graphql

```diff
@@ -893,12 +895,16 @@ type Query {
     @find(model: "User")
     @guard
     @can(ability: "view", query: true, model: "User")
-    @deprecated(reason: "applicant is deprecated. User instead.")
+    @deprecated(
+      reason: "applicant is deprecated. Use User instead. Remove in #7651."
```

Comment:
Another mistake by me, should be `user` lowercase 😬

Reply:
See 30e27dcc7486fcd735b993f0ffcb4add987b5f35.